autoprotocol/protocol.py

```diff
@@ -5135,10 +5141,31 @@ def provision(self, resource_id, dests, volumes):
                 self.instructions[-1].to.append(xfer)
             else:
                 provision_instructions_to_return.append(
-                    self._append_and_return(Provision(resource_id, dest_group))
+                    self._append_and_return(
+                        Provision(resource_id, dest_group, measurement_mode)
+                    )
                 )
         return provision_instructions_to_return
 
+    def _identify_measurement_mode(self, volumes):
```

Comment:
nit: rename `volumes` argument to something more descriptive. It looks like we're expecting a list of `Unit`s. I'd be fine with `amounts` or something like that

Reply:
Renamed `provision_amounts`